service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/RequestStatementMapper.java

```diff
@@ -213,6 +232,26 @@ private String buildTextDescription(ReferralRequest referralRequest) {
         return text.orElse(StringUtils.EMPTY);
     }
 
+    private List<String> buildPractitionersDescription(ReferralRequest referralRequest) {
+        return referralRequest.hasRecipient()
+            ? referralRequest.getRecipient().stream()
```

Comment:
Before each recipient there should be a prefix _**Recipient Practitioner:**_
```
 Recipient Practitioner: { Practitioner.name + " " + Practitioner.family}
```

for example: 
Recipient Practitioner: Arthur George

I am not sure if there would be more how should they be handled?

```
Recipient Practitioner: Arthur George Recipient Practitioner: John Abc
```
or 

```
Recipient Practitioner: Arthur George, John Abc
```
? 
Could you please also add test cases to cover multiple case?

Reply:
Just noticed there seems to be some duplication here with NIAD-916, where various data is being output in sequence without braces. E.g.
`UBRN: bb2378b6-dde2-11e9-9d36-2a2ae2dbcce4 Priority: Routine Specialty: unknown Recipient Org: Organisation name, Recipient Healthcare Service: Consulting psychologists and/or psychology services, Recipient Practitioner: Arthur George Referred for investigation into low blood pressure`

Reply:
I've updated this all to follow the pattern requested in the newer ticket